packages/tupaia-web-server/src/routes/MapOverlaysRoute.ts

```diff
@@ -49,7 +51,7 @@ export class MapOverlaysRoute extends Route<MapOverlaysRequest> {
           comparisonValue: [projectCode],
         },
       },
-      pageSize,
+      pageSize: pageSize || DEFAULT_PAGE_SIZE,
```

Comment:
this is not related to the core issue, but I noticed while testing that the route was only returning a sub set of map overlays so re-instated that default page size here and fixed it